formats/completed_transaction/frontend/examples/completed_transaction.json

```diff
@@ -0,0 +1,54 @@
+{
+  "details": {
+    "external_related_links": [
+
+    ],
+    "presentation_toggles": {
+      "promotion_choice": {
+        "url": "",
+        "choice": "none"
```

Comment:
One of the choices is no promotion, which is required. Hence the choice being 'none'.